playbook.yml

```diff
@@ -146,6 +145,9 @@
       - file_name: 'sonar.hpi'
         checksum: 'sha256:03e2372d36c5bcd7794e0ccafc89bdb552f68ab905f82a55ebfd2e1f201d602f'
         url: 'https://github.com/hmcts/sonarqube-plugin/releases/download/sonar-2.9-hmcts/sonar.hpi'
+      - file_name: 'azure-vm-agents-1.0.2-rc686.884e8965006c.hpi'
```

Comment:
add a comment that this is merged on master pending release and should be updated after 1.0.2 is release

Reply:
done